features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/pages/index.jag

```diff
@@ -44,6 +44,7 @@
 
     <script src="<%= app.context %>/site/public/fonts/iconfont/MaterialIcons.js"></script>
     <script src="<%= app.context %>/site/public/conf/defaultTheme.js"></script>
+    <script src="<%= app.context %>/site/public/conf/portalSettings.js"></script>
```

Comment:
Shall we put a comment here? I think this is a temporary workaround for the `webpack externals` issue ?